src/components/App/index.spec.tsx

```diff
@@ -41,13 +41,26 @@ describe(__filename, () => {
     return root;
   };
 
-  it('renders without crashing', () => {
-    const root = render();
+  it('renders without an authentication token', () => {
+    const root = render({ authToken: '' });
 
     expect(root.find(Container)).toHaveClassName(styles.container);
     expect(root.find(`.${styles.content}`)).toHaveLength(1);
   });
 
+  it('renders with an empty authentication token', () => {
```

Comment:
This test appears to be the same as the one above it.

Reply:
oops